src/main/java/org/folio/rest/converter/costperuse/export/PackageTitlesCostPerUseCollectionToExportConverter.java

```diff
@@ -16,9 +22,22 @@
   @Autowired
   private PackageTitleCostPerUseConverter resourceCostPerUseExportItemConverter;
 
-  public List<TitleExportModel> convert(ResourceCostPerUseCollection resourceCostPerUseCollection, String platform, String year) {
+  private NumberFormat currencyFormatter;
+
+  public List<TitleExportModel> convert(ResourceCostPerUseCollection resourceCostPerUseCollection, String platform, String year, LocaleSettings localeSettings) {
     var data = resourceCostPerUseCollection.getData();
     var currency = resourceCostPerUseCollection.getParameters().getCurrency();
-    return mapItems(data, item -> resourceCostPerUseExportItemConverter.convert(item, platform, year, currency));
+    setLocaleSettings(localeSettings);
+    return mapItems(data, item -> resourceCostPerUseExportItemConverter.convert(item, platform, year, currency, currencyFormatter));
+  }
+
+  private void setLocaleSettings(LocaleSettings localeSettings) {
```

Comment:
I think this method could return `NumberFormat` instead void. In this case there would be no reason to have field `currencyFormatter`

Reply:
usually, setters do not return values, so renamed it to getter and removed field